.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U00002728Enhancement"
+labels: "Type: Enhancement"
+about: Something isn't broken, but should be made better when we have some time.
+
+---
+
+# Enhancement
+
+## What needs done
+
+
+
+## Why it needs done
+
+
+
+## Related issues
```

Comment:
Optional: Consider leaving this out, I think it is already handled more generally with labels and milestones.